src/averageAges.js

```diff
@@ -14,12 +14,31 @@
  *
  * @return {number}
  */
+
+function calculateAveragePeopleAge(people) {
+  const ages = people.map(person => person.died - person.born);
+
+  const sumOfAge = ages.reduce((x, y) => x + y, 0);
+
+  return sumOfAge / ages.length;
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const srtMen = people.filter(human => human.sex === 'm');
+  const artMenCentury = srtMen.filter(person =>
+    Math.ceil(person.died / 100) === century);
+
```

Comment:
what is srtMen?

for now you create 2 arrays for each function call
let's create only one array (call filter method only one time with correct check depending on the century argument)